src/stats/hist.jl

```diff
@@ -81,8 +85,8 @@ function Makie.plot!(plot::Hist)
             return bins
         end
     end
-
     points = lift(edges, plot.normalization, plot.scale_to) do edges, normalization, scale_to
+
```

Comment:
```suggestion

    points = lift(edges, plot.normalization, plot.scale_to) do edges, normalization, scale_to
```

Just to not add unrelated formatting changes to the PR